backend/chainlit/playground/providers/langchain.py

```diff
@@ -65,7 +67,11 @@ async def create_completion(self, request):
 
         messages = self.create_generation(request)
 
-        stream = make_async(self.llm.stream)
+        # https://github.com/langchain-ai/langchain/issues/14980
+        stream = make_async(self.llm.stream)(
```

Comment:
The CI fails because stream is already a coroutine instead of a callable. We can do

```py
        result = await make_async(self.llm.stream)(
            input=messages, **request.generation.settings
        )
```
to replace stream in result in one.